lib/msal-browser/src/utils/BrowserConstants.ts

```diff
@@ -29,6 +25,12 @@ export const BrowserConstants = {
     MSAL_SKU: "msal.js.browser",
 };
 
+export enum BrowserCacheLocation {
```

Comment:
Can we export this from the main package too?

Reply:
Will changing this config value to an enum affect current implementations?

Reply:
today users pass the string "localStorage" or "sessionStorage"

Reply:
I think typescript will recognize that the enum values match those values.

Reply:
done.